raft/src/main/java/org/apache/kafka/snapshot/SnapshotWriter.java

```diff
@@ -113,10 +113,8 @@ private void finalizeSnapshotWithFooter() {
      * Create an instance of this class and initialize
      * the underlying snapshot with {@link SnapshotHeaderRecord}
      *
-     * @param snapshot a lambda to create the low level snapshot writer
```

Comment:
Should this be `supplier` instead of `snapshot`?

Reply:
Good catch!